### ISSUE TITLE
Update vignette

### ISSUE THREAD

**Issue by sdgamboa** [waldronlab/nychanesmicrobiome#9]
Some code stopped working in the smoking.Rmd vignette so I updated the code and created another branch for it https://github.com/waldronlab/nychanesmicrobiome/tree/sdgamboa/update-vignette.

+ The NYC Hanes URL seems to be broken. I used the file included in extdata instead.
+ There was a heatmap only displaying aerobic taxa, but there were no aerobic taxa. Anaerobes and facultive anaerobes are being displayed now.
+ vegan::adonis --> vegan::adonis2. Update code depending on the new format of the output.
+ Minor conversions of sample_data and otu_table to data.frame and matrix, respectively.

**Comment by LiNk-NY** [comment]
Hi @sdgamboa 
Can you create a PR for review? 
Thanks

**Comment by sdgamboa** [comment]
@LiNk-NY, sure. See #10

**Comment by cmirzayi** [comment]
NYCHANES url is still broken but I found a working link here: https://med.nyu.edu/departments-institutes/population-health/divisions-sections-centers/epidemiology/sites/default/files/nyc-hanes-datasets-and-resources-analytic-data-sets-sas-file.sas7bdat

I compared the table 1 published in the original paper vs. the table 1 produced with this dataset and it's identical so I am fairly certain that it's the correct dataset.

From what I can see here are the problems/changes that are needed (will update as I work through the package and I'm working on a pull request to fix everything):

- [ ] Update references to the NYCHANES dataset to point to the NYU link
    - [ ] Possibly put the NYCHANES dataset on Zenodo or somewhere insulated from bitrot (I wasn't sure if this is allowed given that we don't "own" the data and didn't want to do that without permission)
- [ ] Fix lines 208-231 in social-epi.Rmd that use `melt()` which is now defunct I guess.
